example/home/test/test_general.py

```diff
@@ -71,53 +103,95 @@ def test_query_field_plural(self):
             }
         }
         """
-        results = self.client.execute(query)
+        results = self.client.execute(query, context_value=self.request)
         data = results["data"]["posts"]
         self.assertEqual(len(data), 3)
         self.assertEqual(int(data[0]["id"]), self.child_post.id)
         self.assertEqual(int(data[1]["id"]), self.another_post.id)
         self.assertEqual(int(data[2]["id"]), self.blog_post.id)
 
+    def test_query_field_plural_with_user(self):
+        query = """
+        {
+            posts {
+                id
+            }
+        }
+        """
+        self.request.user = AuthenticatedUser()
+        results = self.client.execute(query, context_value=self.request)
+        data = results["data"]["posts"]
+        self.assertEqual(data, None)
```

Comment:
ditto

Reply:
The same applies here regarding the `IsAnonymousMiddleware`.